src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -574,7 +574,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
     }
 
     renderMaterial = (materialList: CDMaterialType[], disableSelection: boolean, isApprovalConfigured: boolean) => {
-        return materialList.map((mat) => {
+        return materialList.map((mat, idx) => {
```

Comment:
this index value will start from 0 & the value you used to get previous i think was unique ids. Please explain in brief around this part